src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Select.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+import React from 'react';
+import MultiSelectComponent from '../../../components/MultiSelect';
+import type {FieldTypeProps} from '../../../types';
+
+const MISSING_VALUES_OPTIONS = 'The "values" option has to be set for the SingleSelect FieldType';
+
+type Props = FieldTypeProps<Array<string | number>>;
+
+export default class Select extends React.Component<Props> {
+    constructor(props: FieldTypeProps<Array<string | number>>) {
+        super(props);
+
+        const {onChange, schemaOptions, value} = this.props;
+
+        if (!schemaOptions) {
+            return;
+        }
+
+        let {
```

Comment:
I would try to leave that as `const`, and introduce another variable below. So that in case the `defaultValues` need to be accessed a second time, it's not a surprise that its content completely changed.